packages/apps/shopify-app-react-router/docs/staticPages/examples/index/auth-route.example.ts

```diff
@@ -0,0 +1,13 @@
+import type {HeadersFunction, LoaderFunctionArgs} from 'react-router';
```

Comment:
Renamed since auth route is more descriptive that splat route.  Splat route for what?